hdx_hapi/endpoints/models/util/constants.py

```diff
@@ -1,4 +1,11 @@
-from pydantic import Field
+from decimal import Decimal
+from typing import Annotated
+from fastapi import Query
+from pydantic import Field, PlainSerializer
 
 
 PERCENTAGE_TYPE = Field(ge=0, le=100)
+
+NON_NEGATIVE_DECIMAL_TYPE = Annotated[
+    Decimal, PlainSerializer(lambda x: float(x), return_type=Annotated[float, Query(ge=0)])
```

Comment:
My reading of this is that pydantic and FastAPI see this as a Decimal but serialize (return in the API) as a float - is that correct?

Reply:
@IanHopkinson , yes, that is correct. Before it was serializing as a string.